src/main/java/tutorial/Bank.java

```diff
@@ -0,0 +1,32 @@
+package tutorial;
+
+public class Bank {
+
+
+//
+    public Bank() {
+
+    }
+
+    public void MortgageProperty(Player player, Properties properties) {
```

Comment:
By naming conventions for methods, the first word is all lowercase so this method should be **m**ortgageProperty.